src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -102,9 +105,45 @@ class YarnBuilder extends AbstractBuilder {
     """
     } catch(ignored) { // TODO remove try catch after pipeline warning expires
       WarningCollector.addPipelineWarning("node_cve", "CVEs found for Node.JS, update your dependencies / ignore false positives", new Date().parse("dd.MM.yyyy", "28.07.2020"))
+    } finally {
+      String issues = steps.readFile('yarn-audit-issues')
```

Comment:
If the reporting itself breaks do we want to block the pipeline? ie. do we want some exception handling in this finally?  Could we log any exceptions here such that we get notified and can investigate?

Reply:
discussed on slack, fine without it for now imo,

We want to know if report preparation fails.

The actual publishing if it fails won't break the build